DFC.ServiceTaxonomy.GraphSync/GraphSyncers/MergeGraphSyncer.cs

```diff
@@ -102,11 +107,11 @@ public class MergeGraphSyncer : IMergeGraphSyncer
             IContentManager contentManager,
             IGraphMergeContext? parentGraphMergeContext = null)
         {
+            _logger.LogDebug("SyncAllowed to {GraphReplicaSetName} for '{ContentItem}' {ContentType}?",
```

Comment:
Another LogDebug here, I'm guessing it's too chatty to turn into a LogInformation and if you're confident it works now, may not be required.